lib/fs-writable.js

```diff
@@ -75,10 +75,11 @@ class WritableFileStream extends Writable {
 
   [errorSymbol](error, callback) {
     if (callback) callback(error);
+    this.emit('error', error);
+
     if (this.autoClose) {
       this.destroy(error);
```

Comment:
Wouldn't the 'error' be emitted twice if `autoClose` is on?
Relevant `destroy` code
https://github.com/metarhia/metastreams/blob/1622a2816305e03577e06300fcf7e7db09d4a2fa/lib/writable.js#L218-L224